code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1812,4 +1812,34 @@
 		var/datum/antagonist/changeling/changeling = L.mind.has_antag_datum(/datum/antagonist/changeling)
 		if(changeling)
 			changeling.chem_charges = max(changeling.chem_charges-2, 0)
-	return ..()
\ No newline at end of file
+	return ..()
+
+/datum/reagent/stasis_brine
+	name = "Stasis Brine"
+	description = "Used inside cloning pods to sustain the growing body."
+	color = "#09999"
+	id = "stasis_brine"
+	metabolization_rate = REAGENTS_METABOLISM
+	self_consuming = TRUE
+
+/datum/reagent/stasis_brine/on_mob_add(mob/M)
+	..()
+	if(isliving(M))
+		var/mob/living/L = M
+		L.add_trait(TRAIT_STABLEHEART, id)
+		L.add_trait(TRAIT_EMOTEMUTE, id)
+		L.add_trait(TRAIT_MUTE, id)
+
+/datum/reagent/stasis_brine/on_mob_delete(mob/M)
+	if(isliving(M))
+		var/mob/living/L = M
+		L.remove_trait(TRAIT_STABLEHEART, id)
+		L.remove_trait(TRAIT_EMOTEMUTE, id)
+		L.remove_trait(TRAIT_MUTE, id)
+	..()
+
+/datum/reagent/stasis_brine/on_mob_life(mob/living/M)
+	M.adjustBruteLoss(-3*REM, 0)
+	M.adjustOxyLoss(-6*REM, 0)
+	M.losebreath = 0
```

Comment:
I mean, if you REALLY want to go all out, have some trait that disables "low health in crit" damage.

Reply:
oh yea i should've thought of that
actually why have a chem at all if i can just use traits all around